libs/server-sdk/src/data_systems/lazy_load/lazy_load_system.hpp

```diff
@@ -110,15 +110,7 @@ class LazyLoad final : public data_interfaces::IDataSystem {
             status_manager_.SetState(DataSourceState::kValid);
 
             if (auto optional_item = *expected_item) {
```

Comment:
Have the transformation from `SerializedItemDescriptor -> StorageItem -> ItemDescriptor` was unnecessarily complex. Now it is back to just `SerializedItemDescriptor -> ItemDescriptor`.